manager/manager/typeresolver_test.go

```diff
@@ -15,14 +15,20 @@ package manager
 
 import (
 	"errors"
+	"fmt"
 	"net/http"
 	"reflect"
+	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/ghodss/yaml"
+	"github.com/kubernetes/deployment-manager/registry"
+	"github.com/kubernetes/deployment-manager/common"
 )
 
+var re = regexp.MustCompile("github.com/(.*)/(.*)/(.*)/(.*):(.*)")
```

Comment:
I don't think so. I made this a static in typeresolver.go but don't know how to use it here.